### PR TITLE
feat: added small variant to cta-image-content component

### LINKED ISSUE #765: Add medium variant to the CtaImageContentLink component
### User story

Add medium variant to CtaImageContentLink as designed: 
https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/Design-System---Gemeente-Den-Haag?node-id=0%3A1

### PR DESCRIPTION
**Pull request by joostfroon**
`closes #765 `

Added small variant to CtaImageContent component.

### CODE REVIEW

File: proprietary/Components/src/denhaag/cta-image-content.tokens.json [hunk 1]
```diff
@@ -17,14 +17,22 @@
         "color": { "value": "{denhaag.color.grey.5}" },
         "font-size": { "value": "2rem" },
         "font-weight": { "value": "700" },
-        "line-height": { "value": "2.563rem" }
+        "line-height": { "value": "2.563rem" },
+        "small": {
+          "font-size": { "value": "1.5rem" },
```
Comment (bddjong) [inline proprietary/Components/src/denhaag/cta-image-content.tokens.json:22]:
```suggestion
          "font-size": { "value": "{denhaag.typography.scale.xl.font-size}" },
```

File: proprietary/Components/src/denhaag/cta-image-content.tokens.json [hunk 2]
```diff
@@ -17,14 +17,22 @@
...
+          "line-height": { "value": "1.938rem" }
```
Comment (bddjong) [inline proprietary/Components/src/denhaag/cta-image-content.tokens.json:23]: I think these line-height values are incorrect. According to the design, the line-height for a small variant is 31.2 pixels. Given a font-size of 24 pixels, that would give a line-height of `1.3`.
```suggestion
          "line-height": { "value": "1.3" }
```

File: proprietary/Components/src/denhaag/cta-image-content.tokens.json [hunk 3]
```diff
@@ -17,14 +17,22 @@
...
       },
       "text": {
         "margin-block": { "value": "0" },
         "color": { "value": "{denhaag.color.grey.4}" },
         "font-weight": { "value": "400" },
         "font-size": { "value": "1.5rem" },
-        "line-height": { "value": "1.938rem" }
+        "line-height": { "value": "1.938rem" },
+        "small": {
+          "font-size": { "value": "1.25rem" },
```
Comment (bddjong) [inline proprietary/Components/src/denhaag/cta-image-content.tokens.json:33]: Same here, try using the denhaag.typography tokens for the scale.

**Review by bddjong — CHANGES_REQUESTED** [review]

File: components/CtaImageContent/src/index.scss [hunk 4]
```diff
@@ -63,9 +71,17 @@
   margin-block-end: var(--denhaag-cta-image-content-text-margin-block);
   color: var(--denhaag-cta-image-content-text-color);
   font-weight: var(--denhaag-cta-image-content-text-font-weight);
-  font-size: var(--denhaag-cta-image-content-text-font-size);
-  line-height: var(--denhaag-cta-image-content-text-line-height);
   font-family: var(--denhaag-typography-sans-serif-font-family);
+
+  &:not(&.denhaag-cta-image-content__text--small) {
```
Comment (mjcarsjens) [inline components/CtaImageContent/src/index.scss:76]: The second `&` (in the `:not()`) is not necessary here, right?

`.denhaag-cta-image-content__text:not(.denhaag-cta-image-content__text--small)` selects exactly the same as the current `.denhaag-cta-image-content__text:not(.denhaag-cta-image-content__text.denhaag-cta-image-content__text--small)`.

**Pull request merged**